### ISSUE TITLE
`aws-iam-authenticator version` returns empty output

### ISSUE THREAD

**Issue by keithlayne** [kubernetes-sigs/aws-iam-authenticator#170]
```
$ go get -u -v github.com/kubernetes-sigs/aws-iam-authenticator/cmd/aws-iam-authenticator                                     
github.com/kubernetes-sigs/aws-iam-authenticator (download)

$ cd $GOPATH/src/github.com/kubernetes-sigs/aws-iam-authenticator && git show 
commit 82544ec86140b7027a07890b09c2f03fca97228a (HEAD -> master, tag: 0.4.0-alpha.1, origin/master, origin/HEAD)
Merge: eb0b44b 96243fb
Author: k8s-ci-robot <k8s-ci-robot@users.noreply.github.com>
Date:   Thu Oct 18 19:40:09 2018 -0700

    Merge pull request #167 from nckturner/github-owner

    Change github repository

$ aws-iam-authenticator version  
{}
```

**Comment by keithlayne** [comment]
Thanks, bot.  I'm glad at least you are paying attention.

/remove-lifecycle-stale

**Comment by nckturner** [comment]
Can you verify this is fixed for you in the newest version with? https://github.com/kubernetes-sigs/aws-iam-authenticator/pull/187

It could probably be improved by using a short git hash with a dirty tag like `git-9999cd80-dirty` if you want to submit a PR.

**Comment by keithlayne** [comment]
Sorry for snarking, I should have checked through issues and PRs when I got pinged by the bot.  I wish this issue was linked in that one before now.

I confirm that #187 works as advertised when run against a version built with `go get`.  Thanks for that!

I think part of the problem (separate issue) is that there is only one place that mentions installation in the README: toward the bottom of [this section](https://github.com/kubernetes-sigs/aws-iam-authenticator#4-set-up-kubectl-to-use-authentication-tokens-provided-by-aws-iam-authenticator-for-kubernetes).  Also, at the time I created this issue, the latest release was pretty far behind master IIRC.  Either way, a prominent section in the readme about installation options showing both the dl/chmod/cp stuff and go get would be helpful IMO.

/close
